be/src/runtime/routine_load/data_consumer.cpp

```diff
@@ -223,6 +223,18 @@ Status KafkaDataConsumer::group_consume(TimedBlockingQueue<RdKafka::Message*>* q
             st = Status::InternalError(ss.str());
             break;
         }
+        case RdKafka::ERR__PARTITION_EOF: {
+            _eof_count++;
+            if (!queue->blocking_put(msg.get())) {
```

Comment:
when partition is EOF, what is the msg.get()? Is there any valid message?

Reply:
And you should explain the logic here to let others understand.

Reply:
And message is added, why put_rows and recept_rows not be accumulated.

Reply:
> when partition is EOF, what is the msg.get()? Is there any valid message?

Is one EOF msg, will be processed when msg is fetched from queue;

Reply:
Fixed